ares/component/processor/mos6502/mos6502.hpp

```diff
@@ -138,6 +183,7 @@ struct MOS6502 {
   n8  S;
   PR  P;
   n16 PC;
+  n16 MAR;
```

Comment:
updates to serialization impact all 6502 based systems; you will need to increase SerializerVersion of both the atari 2600 and famicom cores